templates/partial/_careers-available-roles.html

```diff
@@ -35,6 +35,7 @@
       <hr />
     </div>
   </div>
+  {% if vacancies %}
```

Comment:
Could this go on line 2 instead? If there are no vacancies, there's nothing to filter or sort.

Reply:
@sowasred2012 you are right.